src/slices/azureSlice.js

```diff
@@ -225,14 +225,25 @@ export const importRelease = (
     );
 };
 
-// util
+// utils
```

Comment:
https://nhi.no/sykdommer/barn/autisme/asperger-syndrom/

Reply:
e jo fleire funksjonar som ligg dar for å få lett-leselig kode 
![bilde](https://user-images.githubusercontent.com/26660106/79565626-6036eb00-80b1-11ea-8729-7bb0db01269a.png)

Reply:
Eg va litt slem. Men det bi bære en etter at du har fjerna det vi snakka om.

Reply:
uten denne sjekken oppstår blank tittel problemet. description må være en string for å bli godtatt av api, null vil ikke fungere.

Reply:
Alright